packages/asset-uploader/src/helpers/get-asset-path.ts

```diff
@@ -0,0 +1,27 @@
+import { DbAsset, Location } from "@webstudio-is/prisma-client";
+import { fsEnvVariables, s3EnvVariables } from "../schema";
+import path from "path";
+
+const s3Envs = s3EnvVariables.safeParse(process.env);
```

Comment:
I think this is so fast that you can just do it inside the function and have the trouble with testing